core/core-mcp/src/main/java/ai/wanaku/core/mcp/common/resolvers/Resolver.java

```diff
@@ -1,28 +1,16 @@
 package ai.wanaku.core.mcp.common.resolvers;
 
-import java.io.File;
-
 /**
  * A resolver that consumes MCP requests and resolves what type of tool or resource acquirer
  * should handle it
  */
 public interface Resolver {
-    String DEFAULT_RESOURCES_INDEX_FILE_NAME = "resources.json";
```

Comment:
omg, good catch, I had this stuff in another branch and totally forgot about it!

Thanks, I hope it didn't require a lot of refactor

Reply:
No, it was a pretty quick refactor ... I noticed while working on the MCP forward and fixed it rather quickly.